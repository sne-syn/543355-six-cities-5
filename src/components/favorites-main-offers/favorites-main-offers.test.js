import React from 'react';
import renderer from 'react-test-renderer';
import FavoritesMainOffers from './favorites-main-offers';

jest.mock(`../favorites-list/favorites-list`, () => `FavoritesList`);

test(`Render FavoritesMainOffers`, () => {
  const tree = renderer
    .create(
        <FavoritesMainOffers
          favorites={[]}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
