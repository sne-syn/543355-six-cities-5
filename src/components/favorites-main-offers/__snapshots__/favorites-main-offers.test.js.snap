// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FavoritesMainOffers 1`] = `
<div
  className="page__favorites-container container"
>
  <section
    className="favorites"
  >
    <h1
      className="favorites__title"
    >
      Saved listing
    </h1>
    <FavoritesList
      favorites={Array []}
    />
  </section>
</div>
`;
